src/proto/serializer.ts

```diff
@@ -60,6 +60,11 @@ export class ProtoSerializer {
             protoTransaction.GuardianSignature = transaction.guardianSignature;
         }
 
+        if (transaction.relayer && !transaction.relayer.isEmpty()) {
```

Comment:
Can refactor to have a mini private function `isRelayedTransaction` (for symmetry) .